pure.zsh

```diff
@@ -503,6 +514,10 @@ prompt_pure_async_callback() {
 					;;
 			esac
 			;;
+		prompt_pure_async_renice)
+			# we tried to renice the worker, but if it failed, well, there
+			# is nothing we can do.
```

Comment:
This comment should be removed or reworded. It does not necessarily mean the renice failed, it only indicates that the command completed. Status code is located in `$code`, but it doesn't tell the whole story, e.g. `renice` could've failed, but `ionice` succeeded, which would result in "success".

Reply:
Removed